packages/react-ui/src/app/features/builder/builder-types.ts

```diff
@@ -0,0 +1,106 @@
+import { BlockProperty } from '@openops/blocks-framework';
+import { AiChatContainerSizeState } from '@openops/components/ui';
+import {
+  Flow,
+  FlowOperationRequest,
+  FlowRun,
+  FlowVersion,
+} from '@openops/shared';
+import { DataSelectorSizeState } from './data-selector/data-selector-size-togglers';
+
+export enum LeftSideBarType {
```

Comment:
all types moved from builder-hooks & updated other files to reference this. We get circular dependency issue on build if we don't extract out the types